doc/optimizers.md

```diff
@@ -132,7 +132,7 @@ with _`UpdateRule`_` = AdamUpdate`.
 | `bool` | **`resetPolicy`** | If true, parameters are reset before every Optimize call; otherwise, their values are retained. | `true` |
 
 The attributes of the optimizer may also be modified via the member methods
-`StepSize()`, `BatchSize()`, `Beta1()`, `Beta2()`, `Eps()`, `MaxIterations()`,
+`StepSize()`, `BatchSize()`, `Beta1()`, `Beta2()`, `Eps()`, `MaxIterations()`, `UpdatePolicy()`,
```

Comment:
To be honest `UpdatePolicy()` is not a method that typical users playing with `Adam` should ever need to use, so I would say that we should remove this from the documentation.  Advanced users can take a look inside the code to see what it does more closely and it's well-documented so they shouldn't have a problem there.